<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newAccountDialog</class>
 <widget class="QDialog" name="newAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Account</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/assets/images/icon.svg</normalon>
   </iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>450</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="createAccountTabs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>881</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Random Seed</string>
    </attribute>
    <widget class="QTextEdit" name="privateKeyRandom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>861</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Private Key</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generateRandomButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>391</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate New Account</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addressRandom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>861</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Seed Words</string>
    </attribute>
    <widget class="QTextEdit" name="seedWords">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>861</width>
       <height>70</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Seed Words</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generateWordsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>861</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate New Words</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Private Key</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="privateKeySeed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>861</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addressSeed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>861</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>861</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>You can create a new account using a completely random seed, or generate(or import) an account using random seed words.  Random is more secure, but you will not be able to recreate the account using seed words if you lose your private key.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="accountGenerationProgress">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>691</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="assets/assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newAccountDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newAccountDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
